### ISSUE TITLE
[Spotlight] Alice "Passing through the looking glass"

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#105]
**URL:** https://spotlight.library.tamu.edu/spotlight/alice-in-wonderland/catalog/5b70d3ec22bb9e0aa3832b934886fc22
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.4.3 Contrast Minimum (AA)
**Issues:**  Contrast Error
**Notes:**  Digital collections nav bar selected item text doesn't have sufficient contrast

**Comment by qtamu** [comment]
This contrast combination meets minimum WCAG A standard per: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF80&bcolor=000000

https://marcomm.tamu.edu/our-brand/visual-style/web-branding/web-color-palette/
